Add `pyopenssl` to list of pip3 dependencies

achim-k opened the pull request:
Fixes #515

Unfortunately I did not have time to test this

christophebedard commented:
Thanks for the PR @achim-k!

It would be great if you could test it (`achim-k/setup-ros@patch-1`). Note that you'll need to [install npm](https://github.com/ros-tooling/setup-ros/blob/master/DEVELOPING.md#developing-setup-ros) and [generate build artifacts (i.e., a JS file)](https://github.com/ros-tooling/setup-ros/blob/master/DEVELOPING.md#build-and-test) in order for your change to have an effect.

achim-k commented:
> It would be great if you could test it (`achim-k/setup-ros@patch-1`). Note that you'll need to [install npm](https://github.com/ros-tooling/setup-ros/blob/master/DEVELOPING.md#developing-setup-ros) and [generate build artifacts (i.e., a JS file)](https://github.com/ros-tooling/setup-ros/blob/master/DEVELOPING.md#build-and-test) in order for your change to have an effect.

Done :heavy_check_mark: 
I have added the generated build artifact and squashed the commits into one.

I tested the change in https://github.com/achim-k/rosbridge_suite/commit/b951218af5e113e67779cda4c6a123d72df2fd9f and the job that previously failed succeeds now :+1:

achim-k commented:
@emersonknapp / @christophebedard, would you mind to create a new release now that this PR has been merged?

christophebedard commented:
@achim-k done! https://github.com/ros-tooling/setup-ros/releases/tag/0.4.1
